### PR TITLE
Functions for Inventory of Missing Data

### PR DESCRIPTION
**Pull request by lilyclements**
@dannyparsons this is ready to review

### CODE REVIEW

File: R/plot_inventory_data.R [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+#' plot_inventory_data(data = daily_niger, y = "percentage",
+#'                     elements = c("tmin", "tmax"), display_perc = TRUE,
+#'                     display_count = TRUE)
+
+plot_inventory_data <- function(data, station = NULL, elements, y = c("count", "percentage"),
+                                columns = c("elements", "stations"),
+                                fill = c("none", "stations", "elements"),
+                                facets = c("stations", "elements", "none"),
+                                display_perc = FALSE, display_count = FALSE){
+  checkmate::assert_logical(display_perc)
```
Comment (dannyparsons) [inline R/plot_inventory_data.R:37]: Add `assert_dataframe` for `data`?

File: R/plot_inventory_data.R [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  checkmate::assert_logical(display_count)
+  y <- match.arg(y)
+  columns <- match.arg(columns)
+  fill <- match.arg(fill)
+  facets <- match.arg(facets)
+
+  if ((facets == "stations") & is.null(station)){
+    warning("facets set to none since no stations are given in data")
+    facets = "none"
```
Comment (dannyparsons) [inline R/plot_inventory_data.R:46]:
```suggestion
    facets <- "none"
```

File: R/plot_inventory_data.R [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+  }
+  if ((columns == "stations") & is.null(station)){
+    warning("columns set to `elements` since no stations are given in data")
+    columns = "elements"
```
Comment (dannyparsons) [inline R/plot_inventory_data.R:50]:
```suggestion
    columns <- "elements"
```

File: R/plot_inventory_data.R [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+  }
+  if ((fill == "stations") & is.null(station)){
+    warning("fill set to `none` since no stations are given in data")
+    fill = "none"
```
Comment (dannyparsons) [inline R/plot_inventory_data.R:54]:
```suggestion
    fill <- "none"
```

**Review by dannyparsons — COMMENTED** [review]
Looks good! Couple of small points.

File: R/plot_inventory_data.R (hunk 1 above)
Comment (lilyclements) [reply R/plot_inventory_data.R:37]: I’ve done that in ‘ summarise_inventory_data’ which this function calls

**Review by dannyparsons — APPROVED** [review]

**Pull request merged**